src/test/java/uk/ac/ebi/eva/pipeline/configuration/validation/ParametersValidatorUtilTest.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.configuration.validation;
+
+import org.junit.Test;
+import org.springframework.batch.core.JobParametersInvalidException;
+
+import uk.ac.ebi.eva.pipeline.configuration.validation.step.VepAnnotationGeneratorStepParametersValidatorTest;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Test for {@link ParametersValidatorUtil}
+ */
+public class ParametersValidatorUtilTest {
+
+    @Test
+    public void validString() throws JobParametersInvalidException {
+        ParametersValidatorUtil.checkNullOrEmptyString("any string", "any job parameters name");
+    }
+
+    @Test(expected = JobParametersInvalidException.class)
+    public void emptyString() throws JobParametersInvalidException {
+        ParametersValidatorUtil.checkNullOrEmptyString("", "any job parameters name");
+    }
+
+    @Test(expected = JobParametersInvalidException.class)
+    public void spaceString() throws JobParametersInvalidException {
+        ParametersValidatorUtil.checkNullOrEmptyString(" ", "any job parameters name");
+    }
+
+    @Test(expected = JobParametersInvalidException.class)
+    public void nullString() throws JobParametersInvalidException {
+        ParametersValidatorUtil.checkNullOrEmptyString(null, "any job parameters name");
+    }
+
+
+    @Test
+    public void validBooleanString() throws JobParametersInvalidException {
+        ParametersValidatorUtil.checkBooleanStringSyntax("false", "any job parameters name");
```

Comment:
Should test multiple valid values, `true` and `false` capitalized in different ways.